File: templates/test2.py
import MySQLdb

connArgs={"host": "127.0.0.1", "user": "root", "db": "test"}
conn = MySQLdb.connect(**connArgs)
conn2 = MySQLdb.connect(**connArgs)
conn.autocommit = True
conn2.autocommit = True
cursor = conn.cursor(MySQLdb.cursors.DictCursor)
cursor2 = conn2.cursor(MySQLdb.cursors.DictCursor)
cursor.execute("create table t1 (i int) engine=InnoDB")
cursor.execute("insert into t1 values (1)")
cursor.execute("select * from t1")
cursor2.execute("select * from t1")
conn.commit()
cursor2.execute("select * from t1")

File: templates/test2.pl
use DBD::mysql;
use Carp;

my $dbh = DBI->connect("dbi:mysql:host=127.0.0.1:port=3307:user=root:database=test", undef, undef, { RaiseError => 1 } );
foreach (1..1) {
	my $sth = $dbh->prepare("select UNIX_TIMESTAMP(my_timestamp_field) AS my_name FROM my_table WHERE my_column=4");
    $sth->execute();
    $sth->fetchrow_arrayref;
}
#	$dbh->do("shutdown");
	print "All done.\n";
sleep 10;

